Source/WebCore/css/CSSGroupingRule.cpp

```diff
@@ -110,10 +111,51 @@ ExceptionOr<void> CSSGroupingRule::deleteRule(unsigned index)
     return { };
 }
 
-void CSSGroupingRule::appendCSSTextForItems(StringBuilder& result) const
+String CSSGroupingRule::cssText() const
```

Comment:
It is confusing to call this just `cssText`  as it is a helper that never matches .cssText output. 

It would be more efficient if this took StringBuilder as a parameter and just wrote into it. Maybe `appendCSSTextForChildRulesAndDeclarations` or similar?

Reply:
Changed